exercises/two-fer/two-fer.js

```diff
@@ -0,0 +1,7 @@
+var TwoFer = function () {};
+
+TwoFer.prototype.twoFer = function (who) {
+
```

Comment:
What about adding a comment inside `twoFer`, kind of `// your code goes here` or similar, like in the `hello-world` exercise?